> `pnpm build-misskey-js-with-types`を回すとCIエラーが解消されるはずです（たぶん）

```text
$ pnpm build-misskey-js-with-types

> misskey@2024.2.0-beta.11 build-misskey-js-with-types /home/kisaragi/WebstormProjects/misskey
> pnpm --filter backend build && pnpm --filter backend generate-api-json && ncp packages/backend/built/api.json packages/misskey-js/generator/api.json && pnpm --filter misskey-js update-autogen-code && pnpm --filter misskey-js build && pnpm --filter misskey-js api


> backend@ build /home/kisaragi/WebstormProjects/misskey/packages/backend
> swc src -d built -D

Successfully compiled: 798 files, copied 27 files with swc (167.08ms)

> backend@ generate-api-json /home/kisaragi/WebstormProjects/misskey/packages/backend
> pnpm build && node ./generate_api_json.js


> backend@ build /home/kisaragi/WebstormProjects/misskey/packages/backend
> swc src -d built -D

Successfully compiled: 798 files, copied 27 files with swc (137.58ms)
node:internal/modules/esm/resolve:255
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/kisaragi/WebstormProjects/misskey/packages/backend/node_modules/misskey-reversi/built/esm/index.js' imported from /home/kisaragi/WebstormProjects/misskey/packages/backend/built/core/ReversiService.js
    at finalizeResolution (node:internal/modules/esm/resolve:255:11)
    at moduleResolve (node:internal/modules/esm/resolve:908:10)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///home/kisaragi/WebstormProjects/misskey/packages/backend/node_modules/misskey-reversi/built/esm/index.js'
}

Node.js v20.10.0
/home/kisaragi/WebstormProjects/misskey/packages/backend:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  backend@ generate-api-json: `pnpm build && node ./generate_api_json.js`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.
```

というエラーが出ますね…

--- a/packages/misskey-js/etc/misskey-js.api.md
+++ b/packages/misskey-js/etc/misskey-js.api.md
@@ -2165,7 +2165,7 @@ type IResponse = operations['i']['responses']['200']['content']['application/jso
 type IRevokeTokenRequest = operations['i/revoke-token']['requestBody']['content']['application/json'];
 
 // @public (undocumented)
-function isAPIError(reason: any): reason is APIError;
+function isAPIError(reason: Record<PropertyKey, unknown>): reason is APIError;
 
 // @public (undocumented)
 type ISigninHistoryRequest = operations['i/signin-history']['requestBody']['content']['application/json'];

